README.md

```diff
@@ -69,14 +69,16 @@ Note, that the tests/sanitizers/etc need to be executed from the project root in
 *Supports only clang on MacOS and only gcc on linux*
 
 ### Address/UndefinedBehavior Sanitizers
-`cmake -DENABLE_SANITIZATION=ON` will generate Makefiles with AddressSanitizer and Undefined Behavior options.
+`cmake -ENABLE_ADDR_UB_SANITIZATION=ON` will generate Makefiles with AddressSanitizer and Undefined Behavior options.
 Compile and run them as normal - if any issues are detected, they will be printed to the console.
 It will fail on the first detected error and will print a summary.
 To convert addresses to actual source code locations, make sure llvm-symbolizer is installed (included in the llvm package) and is available in `$PATH`.
 To specify a custom location for the symbolizer, set `$ASAN_SYMBOLIZER_PATH` to the path of the executable.
 This seems to work out of the box on macOS - If not, make sure to have llvm installed.
 The binary can be executed with `LSAN_OPTIONS=suppressions=asan-ignore.txt ./<YourBuildDirectory>/hyriseTest`.
 
+`cmake -ENABLE_THREAD_SANITIZATION=ON` will work as above but with the ThreadSanitizer. It is not possible to combine more than two sanitizers for a single binary.
```

Comment:
That is not exactly true. It depends on what you want to combine. Maybe "Some sanitizers are mutually exclusive, which is why we use two settings for this."